### PR TITLE
Update Course with more content

### PR DESCRIPTION
**Pull request by susannasiebert**
### New Content Checklist

- [ ] New content/chapter is in an Rmd file with [this kind of format and headers](https://github.com/jhudsl/OTTR_Template/blob/main/02-chapter_of_course.Rmd).

- [ ] New content/chapter contains learning objectives.

- [ ] [Bookdown successfully re-renders and any new content files have been added to the _bookdown.yml](https://github.com/jhudsl/OTTR_Template/wiki/Publishing-with-Bookdown).

- [ ] [Spell check runs successfully](https://www.ottrproject.org/customize-robots.html#Spell_checking)).

- [ ] Any newly necessary packages that are needed have been added to the [Dockerfile and image](https://www.ottrproject.org/customize-docker.html).

- [ ] Images are in the [correct format for rendering](https://www.ottrproject.org/writing_content.html#set-up-images).

- [ ] Every new image has [alt text and is in a Google Slide](https://www.ottrproject.org/writing_content.html#Accessibility).

- [ ] Each slide is described in the notes of the slide so learners relying on a screen reader can access the content. See https://lastcallmedia.com/blog/accessible-comics for more guidance on this.

- [ ] The color palette choices of the slide are contrasted in a way that is friendly to those with color vision deficiencies.
You can check this using [Color Oracle](https://colororacle.org/).

### CODE REVIEW

**Review by malachig — APPROVED** [review]
Looking good!

File: 04-outputs.Rmd [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+This filter is currently only run by pVACseq.
+
+### Top Score Filter
+
+The Top Score Filter will attempt to determine the best neoantigen candidate
+for each variants.
+
+For pVACseq it works as follows. Given a set of neoantigen candidates for a
+variant we first group the transcripts into set where all transcripts in a set
```
Comment (tmooney) [inline 04-outputs.Rmd:150]:
```suggestion
variant we first group the transcripts into sets where all transcripts in a set
```

File: 04-outputs.Rmd [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+determine the best neoantigen candidate as follows:
+
+- Pick all neoantigens with a variant transcript that have a protein_coding Biotype
+- Of the remaining candidates, pick the ones with a variant transcript having a
+  TSL less then the `--maximum-transcript-support-level`.
+- Of the remaining candidates, pick the entries with no Problematic Positions
+- Of the remaining candidates, pick the ones passing the Anchor Criteria (explained in
+  more detail further below)
+- Of the remaining candidates, pick the one with the lowest MT IC50 Score (Median or Best
+  depending on the `--top-score-metric`), lowest TSL, and longest transcript.
```
Comment (tmooney) [inline 04-outputs.Rmd:161]: We should be consistent on whether these end with `.`s

File: 04-outputs.Rmd [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+### Determining the Best Transcript and Best Peptide of a Variant
+
+In pVACseq, for each variant, all neoantigen candidates meeting the `--aggregate-inclusion-threshold` are evaluated as follows:
+
+- Pick all entries with a variant transcript that have a protein_coding Biotype
+- Of the remaining entries, pick the ones with a variant transcript having a Transcript Support Level <= `--maximum-transcript-support-level`
+- Of the remaining entries, pick the entries with no Problematic Positions
+- Of the remaining entries, pick the ones passing the Anchor Criteria (see Criteria Details section below)
+- Of the remaining entries, pick the one with the lowest MT IC50 score( Median or Best
+  depending on the `--top-score-metric`), lowest Transcript Support Level, and longest transcript.
```
Comment (tmooney) [inline 04-outputs.Rmd:193]: Similarly here with the `.`s

**Review by tmooney — COMMENTED** [review]
Sorry I'm late to the party! This PR was still open when I started, but I took a bit to get through it 🙂

**Pull request merged**